polyswarm/src/io/polyswarm/app/datamodel/migrations/Migration.java

```diff
@@ -21,15 +21,18 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package io.polyswarm.app.apiclient.v2.requests;
+package io.polyswarm.app.datamodel.migrations;
+
+import java.sql.Connection;
+import java.sql.SQLException;
 
 /**
- * Test results from TestRequest
+ * Interface for defining a migration to the database
+ *
+ * Note for implementers: Make sure your migration can only run once, as we don't have a table to track these
+ *
  */
-public class TestResponse {
-    public final boolean passed;
+public interface Migration {
```

Comment:
I'm not sure why git decided this is a moved file, rather than one deleted, and one new one, but that's not the point of this comment. 

I'm sure there is a project out there that let's people do real migrations in Java with SQLite. I just created this Adhoc interface because the `initializeDatabaseSchema` was getting out of hand. 

If someone knows a real library for creating migrations, let me know.